web/pandas/community/blog/asv-pandas-grant.md

```diff
@@ -0,0 +1,141 @@
+Title: Write up of the NumFOCUS grant to improve pandas benchmarks and diversity
+Date: 2022-03-30
+
+# Write up of the NumFOCUS grant to improve pandas benchmarks and diversity
+
+*By Lucy Jiménez and Dorothy Kabarozi B.*
+
+We want to share our experience working on **Improvements to the**
+**ASV benchmarking framework and diversity efforts** sponsored by
+[NumFOCUS](https://numfocus.org/) to the [pandas](https://pandas.pydata.org/)
+project.
+
+This grant focused on two aspects: the first one is to improve the
+[asv library](https://asv.readthedocs.io/en/stable/), a tool used by
+benchmarking Python packages and used by pandas; this project was
+unmaintained, and the codebase was quite old; additionally, it didn't
+adhere to modern standards, had Python 2 compatibility code that could
+be removed, and also the CI could be improved. The second aspect is
+encouraging more underrepresented groups to contribute to open source
+projects. This grant was held over 10 weeks, working around 20 hours a
+week. It was developed by Dorothy Kabarozi B. from Uganda and Lucy
+Jiménez from Colombia, under the mentoring of Marc Garcia.
+
+## Why were we part of the grant?
+
+Even when we come from different backgrounds, Dorothy from systems
+engineering and Lucy from computational chemistry, we have always been
+interested in participating and contributing to open source software
+projects. For that reason, we have been running the PyLadies meetups in
+our communities ([PyLadies Kampala](https://twitter.com/pyladieskla),
+[PyLadies Colombia](https://bit.ly/sprint-asv)) and have always been on
```

Comment:
Should `PyLadies Colombia` link to https://twitter.com/pyladies_co instead? The bit.ly link seems specific to an event?

Reply:
@mroeschke, thanks for noticing. I'll be changing it!